source/userguide/active_learning.rst

```diff
@@ -0,0 +1,226 @@
+Active Learning in MindMeld
+===========================
+
+Building new conversational applications, adding new domains and intents for existing applications or improving performance based on user experience, all of these benefit from the addition of more informative data to the trained models. Data generation resources and especially user logs for the application are highly beneficial in capturing new trends and growing diversity in terms of the phrasing of queries. While adding both old and new data to the trained models can improve system performance in terms of accuracy, it reduces scalability by increasing the computation and time costs.
```

Comment:
I think this paragraph is a little confusing, since it sounds like we're offering a way of doing data augmentation. 
Consider something more along these lines - 

Once we have a conversational application deployed, capturing user logs are highlight beneficial to understand new trends and diversity in terms of how users are phrasing their queries for different intents. We can manually analyze these logs and annotate them with the correct domain, intent and entity labels that can then be used to train and further improve our models. However, annotation is costly and time consuming. Moreover, as the number of users for the system increase, the number of logs to be annotated also increases by many folds making manually annotation a less scalable process. In such scenarios, we can use Active Learning.